app/controllers/users_controller.rb

```diff
@@ -1,5 +1,10 @@
 class UsersController < ApplicationController
-  def index; end
+  def index
+    @users = User.all
+  end
 
-  def show; end
+  def show
+    @user = User.find(params[:id])
+    @posts = @user.recent_posts
```

Comment:
- [ ] [OPTIONAL] Even though this code works fine when you use `posts` in the view, it's not intuitive, it might confuse someone that you are looping through all available posts, so I would advise you to keep using `@user.recent_posts` in the view so that it will be more intuitive and readable to everyone, in that case, your codes will be more readable and maintainable!